### PR TITLE
Adjust testing.sh for benchmark usage

### PR DESCRIPTION
**Pull request by nyordanoff**
**Description**

This PR adds new flag to the `testing.sh` allowing to specify whether or not to execute benchmark tests. The change is needed because we will use this script from now on for benchmark tests execution in the benchmark job in the test-infra.

Changes proposed in this pull request:
- adapt `testing.sh`
- remove some kyma 1.24.8 leftovers

**Pull Request status**

- [x] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [ ] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: installation/resources/compass-overrides-gke-benchmark.yaml [hunk 1]
```diff
@@ -27,6 +27,8 @@ global:
       manage: true
     regionInstancesCredentials:
       manage: true
+    tenantDependenciesConfig:
```
[the remark this answers is not in the document]
Comment (ivantenevvasilev) [reply installation/resources/compass-overrides-gke-benchmark.yaml:30]: It was merged as part of #2550 in the local overrides and wasn't added to the GKE benchmark overrides, so we added it in this PR.

**Review by PetarTodorovv — APPROVED** [review]

**Pull request merged**